blocks/text/text.css

```diff
@@ -71,6 +71,31 @@ main .block.text.border {
   margin-bottom: 1rem;
 }
 
+main .block.text.rounded-border {
+  padding: 0 2rem;
+  margin-bottom: 1rem;
+}
+
+main .block.text.blue {
+  color: #007CAD;
```

Comment:
This has a token `secondary` in styles.css

Reply:
Can we add this as a general variant on any block. WDYT ?